### PR TITLE
Add hyperhtml@1.12.5

### PR DESCRIPTION
**Pull request by douglasduteil**
hyperHTML looks good :+1: 
I don't know it but I'm curious to see how it will perform 

\cc @WebReflection

### CODE REVIEW

File: hyperhtml-v1.12.5-non-keyed/src/index.js [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+function select(id) {
+  startMeasure("select");
+  store.select(id);
+  app(renderOnMain, store);
+}
+
+function row(state) {
+  const { id, label } = state;
+  const className = classNameSelected(store.selected);
+  return hyper(state, ":row")`
```
Comment (WebReflection) [inline hyperhtml-v1.12.5-non-keyed/src/index.js]: for benchmarks, I'd use `hyper.wire` or `import {bind, wire} from "hyperhtml";` and use bind for the main render and `wire(state, ":row")` here

File: hyperhtml-v1.12.5-non-keyed/src/index.js [hunk 2]
```diff
@@ -0,0 +1,126 @@
...
+import { startMeasure, stopMeasure } from "./utils";
+import { Store } from "./store";
+
+const store = new Store();
+const renderOnMain = hyper(document.querySelector("#main"));
+app(renderOnMain, store);
+
+//
+
+const table = hyper.wire();
```
Comment (WebReflection) [inline hyperhtml-v1.12.5-non-keyed/src/index.js]: apparently you are not using this constant?

**Comment by douglasduteil** [comment]
:up:

**Comment by krausest** [comment]
I'm getting an error (even after locking all versions) - can you please check it:
```
npm run build-prod
npm WARN deprecated babel-preset-es2016@6.24.1: We're super 😸  excited that you're trying to use ES2016 syntax, but instead of making more yearly presets 😭 , Babel now has a better preset that we recommend you use instead: npm install babel-preset-env --save-dev. preset-env without options will compile ES2015+ down to ES5 just like using all the presets together and thus is more future proof. It also allows you to target specific browsers so that Babel can do less work and you can ship native ES2015+ to user 😎 ! We are also in the process of releasing v7, so please give http://babeljs.io/blog/2017/09/12/planning-for-7.0 a read and help test it out in beta! Thanks so much for using Babel 🙏, please give us a follow on Twitter @babeljs for news on Babel, join slack.babeljs.io for discussion/development and help support the project at opencollective.com/babel
npm notice created a lockfile as package-lock.json. You should commit this file.
npm WARN js-framework-benchmark-hyper-html@1.0.0 No repository field.

added 120 packages in 3.862s

> js-framework-benchmark-hyper-html@1.0.0 build-prod /Users/stefan/Source/Javascript/js-framework-benchmark/hyperhtml-v1.12.5-non-keyed
> rollup -c


src/index.js → dist/index.min.js...
[!] Error: 'bind' is not exported by node_modules/hyperhtml/min.js
https://github.com/rollup/rollup/wiki/Troubleshooting#name-is-not-exported-by-module
src/index.js (1:9)
1: import { bind, wire } from "hyperhtml";
            ^
2:
3: import { startMeasure, stopMeasure } from "./utils";

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! js-framework-benchmark-hyper-html@1.0.0 build-prod: `rollup -c`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the js-framework-benchmark-hyper-html@1.0.0 build-prod script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/stefan/.npm/_logs/2017-11-08T19_40_01_026Z-debug.log
```

**Comment by douglasduteil** [comment]
:up: I'm sorry for the merge mess

**Comment by WebReflection** [comment]
you might hate me but hyperHTML version 2 is going to be released today

**Comment by krausest** [comment]
So I'd rather wait for a PR for version 2 and close this one?

**Comment by WebReflection** [comment]
it's out and it's the same, it just needs v2.0.0 instead of v1.....

**Comment by krausest** [comment]
1.12.5 worked now. Then I tried to update to to 2.0.0:
```
src/index.js → dist/index.min.js...
[!] Error: 'default' is not exported by node_modules/hyperhtml/min.js
https://github.com/rollup/rollup/wiki/Troubleshooting#name-is-not-exported-by-module
src/index.js (1:7)
1: import hyperhtml from "hyperhtml";
          ^
2:
3: import { startMeasure, stopMeasure } from "./utils";
```
Tried to change to
`import { bind, wire } from "hyperhtml";` but then I got
```
src/index.js → dist/index.min.js...
[!] Error: 'bind' is not exported by node_modules/hyperhtml/min.js
https://github.com/rollup/rollup/wiki/Troubleshooting#name-is-not-exported-by-module
src/index.js (1:9)
1: import { bind, wire } from "hyperhtml";
            ^
2:
3: import { startMeasure, stopMeasure } from "./utils";
```
Can you take a look at it?

**Comment by WebReflection** [comment]
so, the main is cjs/main.js and the module is esm/main.js ... I don't know what you are loading or how

the cjs/main.js file exports hyperHTML as default

**Comment by WebReflection** [comment]
to clarify: this is wrong `node_modules/hyperhtml/min.js` don't import that one, import the right module/export which is provided by the package.json or import esm/main.js

**Comment by WebReflection** [comment]
I guess 1.12.5 it is then :rofl:

**Comment by krausest** [comment]
That was the strangest merge I've ever made. Guess I pulled it directly into master...

**Comment by WebReflection** [comment]
@krausest for some reason I don't see hyperHTML in the list when I `npm start`

however, version 2.1 went out today and it's a performance oriented release, the only one that makes sense to be in here, I guess ... plus, it's fully native CJS/ESM compatible as module.

Accordingly, what am I supposed to do in order to update this old version of hyperHTML **and** also make it part of the list when anyone does `npm start` ?

**Comment by krausest** [comment]
The index.html is created automatically via npm run index in webdriver-ts.
I added that because I always forgot to add new frameworks by hand. Now I forget to update them via the npm task, sorry :-(
So for the update just rename your hyperHTML directory to reflect the new version, update the files as needed, adjust common.ts in webdriver and execute npm run index.

**Comment by WebReflection** [comment]
this should play better: https://github.com/krausest/js-framework-benchmark/pull/309

thanks a lot

**Pull request merged**